src/entities/fields/api/types.ts

```diff
@@ -0,0 +1,23 @@
+import type { components } from "@/shared/api/generated/schema";
+import { TwinClass_DETAILED } from "@/entities/twinClass";
+import { Featurer_DETAILED } from "@/entities/featurer";
+
+export type Fields = components["schemas"]["TwinClassFieldV2"];
```

Comment:
from now on lets use entities' naming similar to 'schemas'

In this case I advice to rename:
- `src/entities/fields` to `src/entities/twinClassField`
- `export type Fields` to `export type `TwinClassField`
- etc....

❗️ You will find that we already have `src/entities/twinClassField` 
Please use it